common/src/types/v0/message_bus/pool.rs

```diff
@@ -247,15 +247,23 @@ pub struct CreatePool {
     pub id: PoolId,
     /// disk device paths or URIs to be claimed by the pool
     pub disks: Vec<PoolDeviceUri>,
+    /// labels to be set on the pool
+    pub labels: Option<::std::collections::HashMap<String, String>>,
```

Comment:
Rather than using the hash map directly, should we define a new type? i.e. `type PoolLabels = ::std::collections::HashMap<String, String>`. If we did this, we could easily change the type in one place rather than in every place where it is used.